src/Tribe/PUE/Checker.php

```diff
@@ -397,7 +397,7 @@ public function ajax_validate_key() {
 
 				// For multisite, return the network-level siteurl ... in
 				// all other cases return the actual URL being serviced
-				$queryArgs['domain'] = is_multisite() ? $this->get_network_domain() : $_SERVER['SERVER_NAME'];
+				$queryArgs['domain'] = is_multisite() ? $this->get_network_domain() : ( defined( 'WP_SITEURL' ) ? parse_url( WP_SITEURL )['host'] : $_SERVER['SERVER_NAME'] );
```

Comment:
The `parse_url( WP_SITEURL )['host']` is a nice shortcut, but will not play nice with old versions of PHP. Any chance you can bust that up into a few lines so PHP 5.2 doesn't freak out?